arelle/Locale.py

```diff
@@ -10,8 +10,10 @@
 
 (c) Copyright 2011 Mark V Systems Limited, All rights reserved.
 '''
+from __future__ import annotations
 import sys, subprocess
 import regex as re
+from typing import Optional
```

Comment:
Not needed when you're using `from __future__ import annotations`

`Optional[str]` is the same as writing `str | None`

Reply:
Yep missed that one